data/migrations/20190515111950_LinkTable.js

```diff
@@ -0,0 +1,27 @@
+
+exports.up = function(knex, Promise) {
+return knex.schema.createTable('links', function(links){
+links.increments();
+links.integer('event_id');//declaring a foreign key
+links.foreign('event_id')//using the foreign key
```

Comment:
Line 6 is not necessary. The integer event_id is referencing the id from the events table.